Apps/frontend/src/views/Indicators.svelte

```diff
@@ -40,22 +40,22 @@
       const transformedCurrent = transformSampleToYCoord(
         current[channel],
         offsets[channel],
-        scalings[channel]
+        $amplitudeAdjustment[channel]
```

Comment:
Not scope for your Issue, but I think if we move the `amplitude` to the store we should probably also move the `offsets` there. What do you think?

Reply:
Wait, we actually have `offsetAdjustment`, why is `offsets` used here?

Reply:
yea i must have missed to change that